_posts/2016-05-25-explore-moving-filtering.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  "Explore moving filtering"
+date:   2016-05-25 11:09:53 +0100
+categories: Front-end
+banner: ""
+---
+I've never really been a big fan of the refinements menu on our listings page. It is a good way of organising the many many products we have but at times it can be a bit difficult to use.
```

Comment:
can we add <!--more--> after this somewhere, so that we don't see the entire post on the home page. it will cut off at whatever point you put <!--more-->